build-system/tasks/e2e/describes-e2e.js

```diff
@@ -218,25 +259,42 @@ function describeEnv(factory) {
       variants[o.name] = o.value;
     });
 
-    return describeFunc(name, function() {
+    // Use chrome as default if no browser is specified
+    if (!Array.isArray(spec.browsers)) {
+      spec.browsers = ['chrome'];
```

Comment:
What do you think about running tests on all browsers by default?

Reply:
yeah i like that idea (if `browsers` is not specified, run all). i'll measure time increase to run the e2e just so we know what slowdown it adds.

Reply:
so heres a quick check:
gulp e2e --nobuild --testnames --headless  30.84s user 13.94s system 13% cpu 5:25.32 total
gulp e2e --nobuild --testnames --headless  1351.73s user 365.89s system 150% cpu 19:03.70 total

5mins vs 19mins.

i still think its a good idea since obviously we wan't coverage for firefox.

unfortunately there were dozens of resizing errors that only seem to occur on firefox. I'll investigate/fix it on a separate PR and subsequently turn on firefox there too for all tests if i find a global fix.

Reply:
We tried to make the resize logic general, but I have only tested it in Chrome headless and Chrome. We will definitely need to make it work in all browsers, but for now we can run the tests in Chrome only by default and fix the resize logic later. wdyt?